packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -147,7 +143,19 @@ CoreElement createHotRestartButton(Framework framework) {
 
 Future<void> maybeShowDebugWarning(Framework framework) async {
   if (!await serviceManager.connectedApp.isProfileBuild) {
-    framework.showWarning(debugModeWarning);
+    framework.showWarning(children: <CoreElement>[
+      div(
+          text: 'You are running your app in debug mode. Debug mode frame '
+              'rendering times are not indicative of release performance.'),
+      div()
+        ..add(span(
+            text: '''Relaunch your application with the '--profile' switch '''))
+        ..add(a(
+            text: 'or the profile mode option in VS Code or IntelliJ',
```

Comment:
maybe ', or relaunch in profile mode from VS Code or IntelliJ'  ?
The sentence structure doesn't sound parallel to me as is.